release.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
PR looks very good!
However, I'm not sure about this file. Would using repo vars more difficult?

Reply:
Thanks!
I tried to do it with gh commands, but it didn’t work as expected. However, I can try again in another way

Reply:
You need a PAT with these permissions: https://github.com/pagopa/selfcare-onboarding/blob/ab774468aa2573399352d9430a1de788bb24fa13/.github/workflows/create_release_branch.yml#L91

We can use a bot user to create it